REFERENCE.md

```diff
@@ -6,24 +6,24 @@
 
 ### Classes
 
-* [`puppet_agent`](#puppet_agent): == Class: puppet_agent  Upgrades Puppet 4 and newer to the requested version.  === Parameters  [arch]   The package architecture. Defaults to
-* [`puppet_agent::configure`](#puppet_agent--configure)
-* [`puppet_agent::install`](#puppet_agent--install): == Class puppet_agent::install  This class is called from puppet_agent for install.  === Parameters  [version]   The puppet-agent version to 
+* [`puppet_agent`](#puppet_agent): == Class: puppet_agent  Upgrades Puppet 4 and newer to the requested version.  === Parameters    example for clients in dmz:s that need to us
+* [`puppet_agent::configure`](#puppet_agent--configure): This class uses $puppet_agent::config to manage settings in
```

Comment:
This is not a blocking issue for me for this PR, but it seems like Strings is truncating class summaries on line breaks for multiline comments. If it's not a quick fix, could you ticket out fixing this in the future?